tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/Windows/IIS/ApplicationPool.cs

```diff
@@ -3,8 +3,11 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#pragma warning disable SA1108 // BlockStatementsMustNotContainEmbeddedComments
```

Comment:
I "love" that you disabled this instead of just moving the comment inside the catch 😂

Reply:
My comment is at the right place and I'm willing to die on that hill